src/anurag/Assignment29.java

```diff
@@ -0,0 +1,34 @@
+/*Assignment - 29 : 26th Nov'2020 
+Find second maximum from array without sorting the array.*/
+
+package anurag;
+
+public class Assignment29 {
+	
+	void secondHighest(int[] ar) {
+		
+		int highest = ar[0];
+		int secondHighest = 0;
+		for(int i=0; i<ar.length; i++) {
+				
+			if(highest< ar[i]) {
+				highest = ar[i];
+				secondHighest = highest;
+			} 
+			else if(secondHighest<ar[i]) {
+			secondHighest = ar[i];
```

Comment:
you should one extra condition for max number is like

else if (input[index] > secondMax && input[index] != max)